formidable/json_migrations/utils.py

```diff
@@ -41,6 +41,13 @@ def add_fields(ref_fields, new_fields):
 
 def merge_context_forms(forms):
     # forms: role => ContextForm
+
+    # remove version from context and save it in
+    # variable if exists
+    version = None
+    if 'version' in forms:
+        version = forms.pop('version')
```

Comment:
```py
version = forms.pop('version', None)
```

This will do the same in one line. 😉

Reply:
Sure you are right. I also thought about the decorator, but for only one place it seems better like this.

Reply:
Yes, exactly. It looks like a good pattern to implement a contextmanager (when you have to do something before and after), but it'll be a bit overkill here since we are not going to use it somewhere else.
If you want to implement the contextmanager, no problem on my side anyway :wink: